backend/src/main/java/ca/bc/gov/hlth/mohums/controller/MetricsController.java

```diff
@@ -58,6 +59,36 @@ public List<Map<String, Object>> getActiveUserCount() throws SQLException {
         return jdbcTemplate.queryForList(sql);
     }
 
+    @GetMapping("/metrics/total-active-user-count/{format}")
+    public List<Map<String, Object>> getTotalActiveUserCountWeek(@PathVariable String format) throws SQLException {
+        String sqlBase
+                = " SELECT EVENT_DATE, COUNT(1) AS ACTIVE_USER_COUNT"
+                + " FROM ("
+                + "    SELECT TRUNC(to_date('19700101', 'YYYYMMDD') + ( 1 / 24 / 60 / 60 / 1000) * event_time) AS EVENT_DATE"
+                + "    FROM keycloak.event_entity"
+                + "    WHERE type = 'LOGIN'"
+                + "  )"
+                + " %s"
```

Comment:
The way you have implemented this is safe, but please be wary of inserting user-provided data into SQL strings. I realize you are not inserting the user-provided `path` in this code, so the advice is not immediately applicable. 

https://owasp.org/www-community/attacks/SQL_Injection
https://xkcd.com/327/